### DISCUSSION TITLE
ctrl-f bug

### DISCUSSION THREAD

**Discussion by bobrowski** [phcode-dev/phoenix#1998] [General]
so i assume this is a bug, because it wasn't the case in the old brackets.io

i highlight a piece of code and go ctrl-f, fine it searches the file for my selection, i can highlight a new keyword and again go ctrl-f it replaces the search box with the new selection and works as expected

however if i recently did a ctrl-f, highlight something and go ctrl-shift-f then it keeps showing the old selection in the search box

it seems that switching between ctrl-f and ctrl-shift-f and vice versa does not update the contents of the search box with the latest mouse selection/highlight

**Comment by abose** [comment]
Thanks for reporting the issue. This is fixed and will be available in the January release.

Tracking https://github.com/phcode-dev/phoenix/pull/1999
